src/templates/CentralTemplateProvider.ts

```diff
@@ -37,44 +38,44 @@ export class CentralTemplateProvider {
+                const verifiedTemplateIds: string[] = getScriptVerifiedTemplateIds(version).concat(getDotnetVerifiedTemplateIds(version)).concat(getJavaVerifiedTemplateIds());
                 return functionTemplates.filter((t: IFunctionTemplate) => verifiedTemplateIds.find((vt: string) => vt === t.id));
         }
     }
 
-    public async getBindingTemplates(context: IActionContext, projectPath: string | undefined, language: ProjectLanguage, runtime: ProjectRuntime): Promise<IBindingTemplate[]> {
-        const templates: ITemplates = await this.getTemplates(context, projectPath, language, runtime);
+    public async getBindingTemplates(context: IActionContext, projectPath: string | undefined, language: ProjectLanguage, version: FuncVersion): Promise<IBindingTemplate[]> {
+        const templates: ITemplates = await this.getTemplates(context, projectPath, language, version);
         return templates.bindingTemplates;
     }
 
     /**
      * Ensures we only have one task going at a time for refreshing templates
      */
-    private async getTemplates(context: IActionContext, projectPath: string | undefined, language: ProjectLanguage, runtime: ProjectRuntime): Promise<ITemplates> {
+    private async getTemplates(context: IActionContext, projectPath: string | undefined, language: ProjectLanguage, version: FuncVersion): Promise<ITemplates> {
         const providers: TemplateProviderBase[] = [];
         switch (language) {
             case ProjectLanguage.CSharp:
             case ProjectLanguage.FSharp:
-                providers.push(new DotnetTemplateProvider(runtime, projectPath));
+                providers.push(new DotnetTemplateProvider(version, projectPath));
                 break;
             case ProjectLanguage.Java:
-                providers.push(new JavaTemplateProvider(runtime, projectPath));
+                providers.push(new JavaTemplateProvider(version, projectPath));
                 break;
             default:
-                providers.push(new ScriptTemplateProvider(runtime, projectPath));
-                if (runtime !== ProjectRuntime.v1) {
-                    providers.push(new ScriptBundleTemplateProvider(runtime, projectPath));
+                providers.push(new ScriptTemplateProvider(version, projectPath));
+                if (version !== FuncVersion.v1) {
+                    providers.push(new ScriptBundleTemplateProvider(version, projectPath));
                 }
                 break;
         }
 
-        let key: string = runtime;
+        let key: string = version;
         for (const provider of providers) {
             key += provider.templateType;
         }
 
-        context.telemetry.properties.runtime = runtime;
-        context.telemetry.properties.language = language;
+        context.telemetry.properties.projectRuntime = version;
```

Comment:
If you're changing the telemetry anyway, why leave the property name as projectRuntime?

Reply:
A bunch of other cases use `.projectRuntime` for telemetry and I didn't want to change them all. This was the only one using `.runtime` so I only changed it to be consistent